frontend/src/clients/apiClient.ts

```diff
@@ -19,8 +19,26 @@ export interface Species {
   conservationStatus: ConservationStatus;
 }
 
+export interface Sighting {
+  id: number;
+  species: Species;
```

Comment:
Species aren't guaranteed, so try marking this as optional:
```suggestion
  species?: Species;
```
Please double-check this against the backend and see what the exact return value from the backend is when there's no species, so you can handle it properly